kolibri/core/assets/src/core-app/baseClient.js

```diff
@@ -19,6 +19,9 @@ export default function clientFactory(options) {
   client.interceptors.response.use(
     response => response,
     function(error) {
+      if (!error) {
+        error = {};
```

Comment:
don't we have any case in our code using `error === undefined`? if we have it this can cause new problems

Reply:
If we do, it's currently dead code, because this interceptor will error out before it even gets there.